Formula/leafnode.rb

```diff
@@ -14,6 +14,89 @@ def install
                           "--with-user=#{ENV["USER"]}", "--with-group=admin",
                           "--sysconfdir=#{etc}/leafnode", "--with-spooldir=#{var}/spool/news/leafnode"
     system "make", "install"
+    (prefix/"homebrew.mxcl.fetchnews.plist").write fetchnews_plist
+    (prefix/"homebrew.mxcl.texpire.plist").write texpire_plist
+  end
+
+  def caveats; <<-EOS.undent
```

Comment:
No, there is no generic message generated. I did not put `*` as `Homebrew` would probably take care of `leafnode.plist` when using `brew services`. Let me know, if you prefer `*.plist` to selecting of individual files here.

Without custom `caveats` here, `brew info` shows only the following content under `caveats`.
```
==> Caveats
To have launchd start leafnode now and restart at login:
  brew services start leafnode
Or, if you don't want/need a background service you can just run:
  leafnode
```